### PR TITLE
Jira integration

### PR DESCRIPTION
**Pull request by kpelelis**
Added integration with Jira rest API. The use case is to update fix versions once a draft release is initiated. Everything is fully configurable.

### CODE REVIEW

**Comment by kpelelis** [comment]
Bump

File: cmd/draft.go [hunk 1]
```diff
@@ -60,6 +64,28 @@ func draftRelease() error {
 	}
 
 	releaseBody := github.ReleaseBody(diff, viper.GetString("github.release-body-prefix"), branchMap)
+	if updateJiraFixVersions {
+		// Eliminate dublicates by using a map
```
Comment (dbaltas) [inline cmd/draft.go]: typo on duplicates

File: jira/jira.go [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+func NewClient(httpClient *http.Client, baseURL string) (*Client, error) {
+	parsedBaseURL, err := url.Parse(baseURL)
+	if err != nil {
+		return nil, err
+	}
+
+	if !strings.HasSuffix(baseURL, "/") {
+		baseURL += "/"
```
Comment (dbaltas) [inline jira/jira.go:29]: maybe `parsedBaseURL += "/"`?

File: jira/jira.go [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+	}
+
+	res, err := c.client.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	return res, nil
+}
+
+func (c *Client) UpdateIssueFixVersions(tasks []string) {
```
Comment (dbaltas) [inline jira/jira.go]: Similar to `UpdateIssue`, this method should return an error object, which the caller will be responsible to handle.

**Review by dbaltas — COMMENTED** [review]
Interesting and useful stuff!

**Pull request merged**